### PR TITLE
Add new workflow for Dell Racadm Get firmware catalog

### PR DESCRIPTION
**Pull request by uppalk1**
Unit test for /dell-racadm-get-firmware-list-graph

jenkins: depends on https://github.com/RackHD/on-tasks/pull/435

### CODE REVIEW

**Comment by uppalk1** [comment]
depends on :RackHD/on-tasks#435

**Comment by uppalk1** [comment]
test this please

File: spec/lib/graphs/dell-racadm-get-firmware-list-graph-spec.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// Copyright 2016, EMC, Inc.
```
Comment (yyscamper) [inline spec/lib/graphs/dell-racadm-get-firmware-list-graph-spec.js]: incorrect copyright date.

**Comment by uppalk1** [comment]
test this please

File: lib/graphs/dell-racadm-get-firmware-list-graph.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+// Copyright 2017, EMC, Inc
```
Comment (anhou) [inline lib/graphs/dell-racadm-get-firmware-list-graph.js]: Please use the latest copyright statements, the same for other files

**Pull request merged**